requirements.txt

```diff
@@ -1,9 +1,7 @@
-datetime
 fiona
 geopandas
 h5py
 matplotlib
 numpy
-pre-commit
```

Comment:
Just double checking that these two removals are ok? I think the `datetime` removal is fine, but unsure about `pre-commit` which was added in #96.

Reply:
Seems to me the `pre-commit` is more for linting the code, not really for actually functionality.

Reply:
Agree. Might need to list it somewhere else as a development-only dependency though (can be done in a separate PR).

Reply:
Looks like pre-commit is already in requirements-dev.txt, albeit with a typo so it reads pre_commit instead...